trafficgen-server-start

```diff
@@ -127,7 +155,7 @@ echo -e "${CMD_OUTPUT}"
 WORKLOAD_CPUS_SEPARATED=$(echo -e "${CMD_OUTPUT}" | grep "final cpus:" | awk '{ print $3 }' | sed -e "s/,/ /g")
 WORKLOAD_CPUS_ARRAY=(${WORKLOAD_CPUS_SEPARATED})
 
-testpmd_cores=${#WORKLOAD_CPUS_ARRAY[@]}
+testpmd_cores=$(( ${#WORKLOAD_CPUS_ARRAY[@]} - 1 ))
```

Comment:
What is this changing?

Reply:
I forgot to take that out. I was testing with all CPUs for testpmd coming from workload_cpus and no housekeeping, as I wanted to ensure all of testpmd cpus were in the same NUMA node.